src/sentry/tasks/post_process.py

```diff
@@ -411,6 +411,17 @@ def fetch_buffered_group_stats(group):
     group.times_seen_pending = result["times_seen"]
 
 
+def fetch_buffered_group_last_seen(event, group):
```

Comment:
I don't think you can rely on having an event here, when you're serializing `Group` models you probably won't have an event